src/ui.js

```diff
@@ -558,7 +560,7 @@ HanabiCard.prototype.reset = function() {
 HanabiCard.prototype.add_listeners = function() {
 	var self = this;
 
-	this.on("mouseover tap", function() {
+	this.on("mousemove tap", function() {
```

Comment:
I suppose that it would be possible to merge your note handler into this handler now.